openquakeplatform_building_class/models.py

```diff
@@ -65,25 +71,40 @@ class OCCUP_TYPE(object):
 
 
 BOOL_CHOICES = ((True, 'Yes'), (False, 'No'))
+
+
 class UserSettings(SingleOwnerMixin, models.Model):
-    name = models.CharField(_('Individual Name'), max_length=255, blank=True, null=True,
-                            help_text=_('name of the responsible person: surname, given name,'
+    name = models.CharField(_('Individual Name'), max_length=255, blank=True,
+                            null=True,
+                            help_text=_('name of the responsible'
+                                        ' person: surname, given name,'
                                         ' title separated by a delimiter'))
     organization = models.CharField(_('Organization Name'), max_length=255,
                                     blank=True, null=True,
-                                    help_text=_('name of the responsible organization'))
-    position = models.CharField(_('Position Name'), max_length=255, blank=True, null=True,
-                                help_text=_('role or position of the responsible person'))
+                                    help_text=_('name of the responsible'
+                                                ' organization'))
+    position = models.CharField(_('Position Name'), max_length=255, blank=True, 
+                                null=True,
+                                help_text=_('role or position of the'
+                                            ' responsible person'))
     publish_info = models.BooleanField(choices=BOOL_CHOICES, help_text=_('''Would you like your contribution to be acknowledged in future documentation (eg. technical reports, scientific papers in peer reviewed journals, ...) ?<br>
 If you answer <b>no</b>, you will not appear in the contributors list and your contribution will remain anonymous.<br>
           If you answer <b>yes</b>, please let us know how you prefer to be mentioned by adding your credentials'''
     ))
 
+
 class UserSettingsForm(ModelForm):
+    publish_info = forms.TypedChoiceField(
+                       coerce=lambda x: x == 'True',
+                       choices=((False, 'False'),
+                                (True, 'True')),
+                       widget=forms.RadioSelect)
+
     class Meta:
         model = UserSettings
         fields = ['name', 'organization', 'position', 'publish_info']
-        widgets = { 'publish_info': forms.RadioSelect }
+        # widgets = { 'publish_info': forms.RadioSelect }
+
```

Comment:
Remove commented code

Reply:
Fixed whit this 29d81fefc63f9fa1c989eacb84829f2a524b0375